diff --git a/node_modules/eslint-plugin-tailwindcss/lib/rules/classnames-order.js b/node_modules/eslint-plugin-tailwindcss/lib/rules/classnames-order.js
index cd0d2b0..caa4328 100644
--- a/node_modules/eslint-plugin-tailwindcss/lib/rules/classnames-order.js
+++ b/node_modules/eslint-plugin-tailwindcss/lib/rules/classnames-order.js
@@ -147,7 +147,7 @@ module.exports = {
             // The problem is that range computation includes the backticks (`test`)
             // but value.raw does not include them, so there is a mismatch.
             // start/end does not include the backticks, therefore it matches value.raw.
-            const txt = context.getSourceCode().getText(arg);
+            const txt = context.sourceCode.getText(arg);
             prefix = astUtil.getTemplateElementPrefix(txt, originalClassNamesValue);
             suffix = astUtil.getTemplateElementSuffix(txt, originalClassNamesValue);
             originalClassNamesValue = astUtil.getTemplateElementBody(txt, prefix, suffix);
diff --git a/node_modules/eslint-plugin-tailwindcss/lib/rules/enforces-shorthand.js b/node_modules/eslint-plugin-tailwindcss/lib/rules/enforces-shorthand.js
index ebe2f19..8a2ca73 100644
--- a/node_modules/eslint-plugin-tailwindcss/lib/rules/enforces-shorthand.js
+++ b/node_modules/eslint-plugin-tailwindcss/lib/rules/enforces-shorthand.js
@@ -228,7 +228,7 @@ module.exports = {
             // The problem is that range computation includes the backticks (`test`)
             // but value.raw does not include them, so there is a mismatch.
             // start/end does not include the backticks, therefore it matches value.raw.
-            const txt = context.getSourceCode().getText(arg);
+            const txt = context.sourceCode.getText(arg);
             prefix = astUtil.getTemplateElementPrefix(txt, originalClassNamesValue);
             suffix = astUtil.getTemplateElementSuffix(txt, originalClassNamesValue);
             originalClassNamesValue = astUtil.getTemplateElementBody(txt, prefix, suffix);
diff --git a/node_modules/eslint-plugin-tailwindcss/lib/rules/migration-from-tailwind-2.js b/node_modules/eslint-plugin-tailwindcss/lib/rules/migration-from-tailwind-2.js
index 01abb90..044c59b 100644
--- a/node_modules/eslint-plugin-tailwindcss/lib/rules/migration-from-tailwind-2.js
+++ b/node_modules/eslint-plugin-tailwindcss/lib/rules/migration-from-tailwind-2.js
@@ -148,7 +148,7 @@ module.exports = {
             // The problem is that range computation includes the backticks (`test`)
             // but value.raw does not include them, so there is a mismatch.
             // start/end does not include the backticks, therefore it matches value.raw.
-            const txt = context.getSourceCode().getText(arg);
+            const txt = context.sourceCode.getText(arg);
             prefix = astUtil.getTemplateElementPrefix(txt, originalClassNamesValue);
             suffix = astUtil.getTemplateElementSuffix(txt, originalClassNamesValue);
             originalClassNamesValue = astUtil.getTemplateElementBody(txt, prefix, suffix);
